docs/dev/tools/cli.md

```diff
@@ -1231,7 +1250,7 @@ viam machines part restart --machine=123 --part=456
 | -------- | ----------- | ------------------- | --------- |
 | `--organization` | Organization name or ID that the machine belongs to | `list`, `status`, `logs`, `part` | **Required** |
 | `--location` | Location name or ID that the machine belongs to or to list machines in | `list`, `status`, `logs`, `part` | **Required** |
-| `--machine` | Machine name or ID for which the command is being issued | `status`, `logs`, `part`, `part restart` | **Required** |
+| `--machine` | Machine name or ID for which the command is being issued | `status`, `logs`, `part` | **Required** |
```

Comment:
```suggestion
| `--machine` | Machine name or ID for which the command is being issued. If machine name is used instead of ID, `--organization` and `--location` are required | `status`, `logs`, `part` | **Required** |
```
I think based on Slack thread? Maybe only applies to `part cp` though so disregard if so